angular/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "angular",
+  "version": "0.0.0",
+  "scripts": {
+    "ng": "ng",
+    "start": "ng serve",
+    "build": "ng build",
+    "test": "ng test",
+    "lint": "ng lint",
+    "e2e": "ng e2e"
+  },
+  "private": true,
+  "dependencies": {
+    "@angular/animations": "~9.1.13",
```

Comment:
Imagekit angular to be published 2.0.0 version to be added here.

Reply:
Added